inc/flexible-cpts.php

```diff
@@ -540,6 +549,31 @@ function hale_get_flexible_post_type_object_type($post_type){
 
 }
 
+/**
+ * Retrieves the settings for a given post type.
+ *
+ * @param string $post_type The post type key.
+ * @return array The object type.
+ */
+function hale_get_flexible_post_type_settings($post_type){
+    $cpts = get_field( 'custom_post_types', 'options' );
+
+    $flex_cpt = false;
+
+    if(!empty($cpts) && is_array($cpts)){
+        foreach($cpts as $cpt){    
+            $post_type_key = $cpt['post_type_key'];
+            if($post_type_key == $post_type){
+                $flex_cpt = $cpt;
+            }
```

Comment:
Do you want to break the loop here, so it doesn't continue looping once `$post_type_key == $post_type`?